tracer/test/Datadog.Trace.Security.Unit.Tests/IAST/Tainted/vulnerability_schema.json

```diff
@@ -67,6 +67,8 @@
             "type": "string",
             "enum": [
               "COMMAND_INJECTION",
+              "DIRECTORY_LISTING_LEAK",
+              "HARDCODED_PASSWORD",
```

Comment:
Is `HARDCODED_PASSWORD` meant to have been added?

Reply:
I synced the [vulnerability_schema.json](https://github.com/DataDog/experimental/blob/main/teams/asm/vulnerability_schema/vulnerability_schema.json)

The sync that adds the `HARDCODED_PASSWORD`, has been forgotten in a previous PR.